src/components/menu/menu-item/menu-item.component.js

```diff
@@ -70,14 +256,17 @@ const MenuItem = ({
 
   return (
     <StyledMenuItemWrapper
+      { ...ifRef() }
```

Comment:
Why is this conditional?

Reply:
I need to have the conditional here because if there is a submenu prop we need to push the ref deeper to another one and also we don't want to put ref in a div element